aries-site/src/pages/components/layer/center-layer.mdx

```diff
@@ -0,0 +1,69 @@
+import { Notification } from 'grommet';
+import { Example } from '../../../layouts';
+import {
+  CenterInformational,
+  CenterLayerAnatomy,
+  ContentDrivenLayout,
+} from '../../../examples';
+
+## Anatomy
+
+This diagram highlights details specific to center layers. See [general layer anatomy](/components/layer#anatomy) for specifications applying to all layers.
+
+<Example plain>
+  <CenterLayerAnatomy />
+</Example>
+
+| Label | Region             | Purpose                                                          | Required?                                  | Notes                                                                                                                                                                                                                                |
+| ----- | ------------------ | ---------------------------------------------------------------- | ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Close button**   | Closes the layer.                                                | Required on layers with no footer actions. | Use "FormClose" icon.                                                                                                                                                                                                                |
+| **2** | **Footer actions** | Allows the user to confirm, apply, or cancel the layer's action. | Required on actionable layers.             | The actions should be right-aligned with primary action on the right. Right-alignment signals progress and forward momentum. One action should be a "Cancel" button that closes the layer, returning the user to the previous state. |
+
+See [layer closing behavior](/components/layer#layer-closing-behavior) for details.
+
+<Notification
+  status="info"
+  message="A layer should either have actions in the footer or a close button in the header, not both."
```

Comment:
I wonder if users will miss this from looking at the Anatomy since it shows both should we have a Do and Dont to relate to this? Ive just seen in the past people were still using a `close button` and `X` in header